components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt

```diff
@@ -18,3 +18,15 @@ fun String.toNormalizedUrl(): String {
     }
     return uri.toString()
 }
+
+/**
+ * Checks if this string is a URL.
+ */
+fun String.isUrl(): Boolean {
+    val trimmedUrl = this.trim { it <= ' ' }
```

Comment:
There's a trim() implementation for Strings that doesn't require a lambda if you want to remove whitespaces:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim.html